Components/DayCalendar.js

```diff
@@ -9,50 +12,251 @@ var {
   ScrollView,
   View,
   Text,
+  PanResponder,
+  Image,
 } = React;
 
+type Event = {
+  title: string;
+  startAt: Date;
+  length: number;
+};
+
 var HOURS_COUNT = 25;
 var HOURS = [];
 for (var i = 0; i < HOURS_COUNT; i++) {
-    var hour = moment().hour(i).second(0).minute(0).format('h:mma');
+    var hour = moment().hour(i).minute(0).second(0).format('h:mma');
     HOURS.push(hour);
 }
+var MINUTES_INA_DAY = 60*24;
+
+var DayCalendar = React.createClass({
+
+  panResponder: {},
+  resizeResponder: {},
+  draggableViewPreviousTop: 0,
+  draggableViewPreviousHeight: 0,
+  draggableView: (null : ?{ setNativeProps(props: Object): void }),
+  resizeHandleView: (null : ?{ setNativeProps(props: Object): void }),
```

Comment:
This is flowtype annotation. Annotaion like `name: string` is easy to understand. But this one is very special. I haven't found any doc that explains it. I copied it from [this example](https://github.com/facebook/react-native/blob/master/Examples/UIExplorer/ResponderExample.js). My current understanding is that it means "draggableView has a setNativeProps method".